google/cloud/storage/CMakeLists.txt

```diff
@@ -326,14 +329,14 @@ install(TARGETS storage_common_options nlohmann_json
 
 # Export the CMake targets to make it easy to create configuration files.
 install(EXPORT storage-targets
-        DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/storage_client")
+        DESTINATION "${CMAKE_INSTALL_DATAROOTDIR}/cmake/Modules")
 
 install(TARGETS storage_client
         EXPORT storage-targets
         RUNTIME DESTINATION bin
         LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
         ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
-install(DIRECTORY . DESTINATION include/storage/client
+install(DIRECTORY . DESTINATION include/google/cloud/storage/client
```

Comment:
This was (and still is) the wrong directory to install the files. I think this is only working because the CMake list in `google/cloud` installs the headers anyway.  No action needed from you, just a comment, filed #1179 to fix that.